src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +344,40 @@ Demos
 
 ## FAQ
 
+**Do I need the `--disable-gpu` flag?**
+
+Yes, for now.  The `--disable-gpu` flag is a temporary requirement to work around a few bugs. You won't need this
+flag in future versions of Chrome. See [https://crbug.com/546953#c152](https://bugs.chromium.org/p/chromium/issues/detail?id=546953#c152) and [https://crbug.com/695212](https://bugs.chromium.org/p/chromium/issues/detail?id=695212) for more information.
+
+**So I still need Xvfb?**
```

Comment:
You can make this a subsection of `Starting Headless`. I.e. on line 62:

`### Do I still need Xvfb?`

Or you could just make this a `note` at the bottom of `Starting Headless`